modules/modal/react/lib/Modal.tsx

```diff
@@ -10,12 +11,36 @@ export enum ModalWidth {
 }
 
 export interface ModalProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * This will add a `data-testid` attribute to the Popup component, not the container
+   */
+  testId?: string;
   open: boolean;
-  padding: PopupPadding;
-  transformOrigin: TransformOrigin;
-  width: ModalWidth;
+  padding?: PopupPadding;
+  width?: ModalWidth;
+  /**
+   * Optional callback for the Modal handling closing. If this callback is provided
+   */
   handleClose?: () => void;
-  heading?: React.ReactNode;
+  /**
+   * Accessibility specifications state modals should be close when the escape key is pressed.
```

Comment:
Nit: `close` > closed

Reply:
It may also be worth calling out the use case for a UI blocking modal. That's not immediately clear when reading this.